### DISCUSSION TITLE
Still having issues with resolution picking up reliably.

### DISCUSSION THREAD

**Discussion by adam-rudd** [meisnate12/Plex-Meta-Manager#2004] [General]
Log:  
[meta.log](https://github.com/Kometa-Team/Kometa/files/15112441/meta.log)

Take Hateful Eight as below, just keeps saying:  
> | The Hateful Eight                                                                                  |
|   Overlay Update Not Needed      

![](https://i.imgur.com/STORJbc.png)

![](https://i.imgur.com/7NHvROO.png)

**Comment by chazlarson** [comment]
If you enable trace logging, Kometa will tell you specifically why it thinks:
```
Overlay Update Not Needed
```
I suspect it's falling into one of the overlays you have turned off.  I would probably do a run with nothing disabled to see what overlay it gets in that case, then work from there.
